### ISSUE TITLE
Add an Argument to Control the Number of Data Samples Used for Parameter Growth in FedTinyClean

### ISSUE THREAD

**Issue by Little0o0** [FedPruning/FedPruning#10]
In the current FedTinyClean method, the entire training dataset is used to calculate gradients for weight growth. You can refer to the relevant code in ```api/standalone/fedtinyclean/my_model_trainer_classification.py``` Line 64-70
```
if mode in [2, 3]:
    for batch_idx, (x, labels) in enumerate(train_data):
        x, labels = x.to(device), labels.to(device)
        log_probs = model(x)
        loss = criterion(log_probs, labels)
        loss.backward()
```
Please add an argument to control the number of data samples used for parameter growth in FedTinyClean. Options should include no data (random growth), one data, one batch of data, or the entire dataset.
